pkg/virt-operator/creation/rbac/handler.go

```diff
@@ -99,22 +99,6 @@ func newHandlerClusterRole() *rbacv1.ClusterRole {
 				},
 				Verbs: []string{
 					"patch",
-					"list",
-					"watch",
-					"get",
-				},
-			},
-			{
-				APIGroups: []string{
-					"",
-				},
-				Resources: []string{
-					"configmaps",
```

Comment:
You will have to keep all configmap logic in the  operator to allow people upgrading. Just remove the configmap.

Reply:
Updated